src/lib/l10n/l10n-singleton.js

```diff
@@ -1,3 +1,4 @@
+/* istanbul ignore file */

```

Comment:
what is this comment?

Reply:
This comment removes this file from jest coverage - I removed L10n files from coverage - as it won't be tested